include/Reader/readerir.h

```diff
@@ -794,7 +794,7 @@ class GenIR : public ReaderBase {
   void callMonitorHelper(bool IsEnter);
 
   IRNode *convertToBoxHelperArgumentType(IRNode *Opr,
-                                         CorInfoType CorType) override;
+                                         uint32_t DestSize) override;
```

Comment:
Nit: can you make the name of the last param the same in reader.h and readerir.h? They are Size and DestSize.